test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -189,6 +195,8 @@ class cursor_bound_01 : public test {
             }
             return;
         }
+        if (range_ret == WT_ROLLBACK)
```

Comment:
We should probably do this check before, where we check `normal_ret == WT_ROLLBACK`.

Reply:
e46764d